pkgs/tools/security/scorecard/default.nix

```diff
@@ -56,6 +56,9 @@ buildGoModule rec {
     }
     # Ensure other e2e tests that have escaped the e2e dir dont run
     export SKIP_GINKGO=1
+
+    # TODO
```

Comment:
Would `checkFlags` work in this case?

See pkgs/tools/admin/aws-sso-cli/default.nix for an example

Also if the comment could be updated that'd be great